### PR TITLE
Added GitHub loading spinner

### PR DESCRIPTION
**Pull request by dnlvgl**
Added the Github loading spinner as an option with `data-gist-show-spinner="true"`

### CODE REVIEW

File: gist-embed.js [hunk 1]
```diff
@@ -47,6 +48,7 @@
       hideLineNumbersOption = $elem.data('gist-hide-line-numbers') === true;
       lines = $elem.data('gist-line');
       highlightLines = $elem.data('gist-highlight-line');
+      showSpinner = $elem.data('gist-show-spinner') === true;
       showLoading = $elem.data('gist-show-loading') !== undefined ?
```
Comment (blairvanderhoof) [inline gist-embed.js]: showLoading should be false if showSpinner is true so we don't have text and images?

File: gist-embed.js [hunk 2]
```diff
@@ -63,6 +65,11 @@
       if (showLoading) {
         $elem.html('Loading gist ' + url + (data.file ? ', file: ' + data.file : '') + '...');
       }
+    
+      // loading spinner
+      if (showSpinner) {
+        $elem.html('<img style="display:block;margin-left:auto;margin-right:auto" src="https://assets-cdn.github.com/images/spinners/octocat-spinner-32.gif">');
```
Comment (blairvanderhoof) [inline gist-embed.js]: Let's add an alt tag to the image with this value https://github.com/blairvanderhoof/gist-embed/blob/master/gist-embed.js#L64, make that a variable so you can reuse it in both places.

**Comment by blairvanderhoof** [comment]
Thanks for doing this! Can you provide screenshots?

**Comment by dnlvgl** [comment]
Screenshot:

**Comment by shivapoudel** [comment]
@dnlvgl and @blairvanderhoof if the Github spinner is loaded then `showLoading` should be disable :)

**Comment by dnlvgl** [comment]
@blairvanderhoof fixed both issues.

File: gist-embed.js [hunk 3]
```diff
@@ -47,9 +48,14 @@
       hideLineNumbersOption = $elem.data('gist-hide-line-numbers') === true;
       lines = $elem.data('gist-line');
       highlightLines = $elem.data('gist-highlight-line');
+      showSpinner = $elem.data('gist-show-spinner') === true;
+      if (showSpinner) {
+        $elem.data('gist-show-loading') === false;
```
Comment (blairvanderhoof) [inline gist-embed.js]: This is just checking equality not an assignment. But better yet, can you do this?

````javascript
if (showSpinner) {
  showLoading = false;
} else {
  showLoading = $elem.data('gist-show-loading') !== undefined ?
    $elem.data('gist-show-loading') : true;
}

File: gist-embed.js [hunk 4]
```diff
@@ -47,9 +48,14 @@
...
+      }
       showLoading = $elem.data('gist-show-loading') !== undefined ?
         $elem.data('gist-show-loading') : true;
 
+
```
Comment (blairvanderhoof) [inline gist-embed.js]: Remove the extra line

**Comment by blairvanderhoof** [comment]
Looks good, just a couple last things and we are good to go.

**Comment by dnlvgl** [comment]
@blairvanderhoof all done.

**Pull request merged**